src/DynamoCore/Logging/AnalyticsService.cs

```diff
@@ -11,6 +13,20 @@ namespace Dynamo.Logging
     /// </summary>
     class AnalyticsService
     {
+        private static readonly ManualResetEventSlim waitInit = new ManualResetEventSlim(false);
+        static AnalyticsService()
+        {
+            // Initialize the concrete class only when we initialize the Service.
+            // This will also load the Analytics.Net.ADP assembly
+            // We must initialize the ADPAnalyticsUI instance before the Analytics.Start call.
+            Task.Run(() =>
```

Comment:
we have a DisableAnalytics flag which (when set to true) is supposed to disable all network activity (and actually not even load ADP binaries). The way this code is written now, looks like it will call into ADP regardless

Reply:
We should really add that automated test for network activity eh? 😉